enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/EnterpriseAuthManagerFactory.java

```diff
@@ -43,9 +50,30 @@ public EnterpriseAuthManagerFactory()
     @Override
     public AuthManager newInstance( Config config, LogProvider logProvider )
     {
+        // Resolve auth store file names
+        File authStoreDir = config.get( GraphDatabaseSettings.auth_store_dir );
+        File userStoreFile;
+        if ( authStoreDir != null )
+        {
+            userStoreFile = new File( authStoreDir, USER_STORE_FILENAME );
+        }
+        else
+        {
+            // Fallback on the directory of the legacy setting
+            userStoreFile = config.get( GraphDatabaseSettings.auth_store );
```

Comment:
Should we support legacy settings? Perhaps since this is new code, we should not. Or do we have an intermediate state where current configs have no default?